spec/forms/candidate_interface/immigration_route_form_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::ImmigrationRouteForm, type: :model do
+  describe '#validations' do
```

Comment:
seems like this is all wrapped in this block and we need a test or two for validations